nemo_curator/utils/script_utils.py

```diff
@@ -165,6 +165,75 @@ def parse_gpu_dedup_args(
     return parser
 
 
+def parse_distributed_classifier_args(
+    description="Default distributed classifier argument parser",
+) -> argparse.ArgumentParser:
+    """
+    Adds default set of arguments that are common to multiple stages
+    of the pipeline
+    """
+
+    parser = argparse.ArgumentParser(
+        description,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser = add_distributed_args(parser)
+    # Set low default RMM pool size for classifier
+    # to allow pytorch to grow its memory usage
+    # by default
+    parser.set_defaults(rmm_pool_size="512MB")
+    parser.add_argument(
+        "--input-data-dir",
+        type=str,
+        help="The path of the input files",
+        required=True,
+    )
+    parser.add_argument(
+        "--output-data-dir",
+        type=str,
+        help="The path of the output files",
+        required=True,
+    )
+    parser.add_argument(
+        "--model-path",
+        type=str,
+        help="The path to the model file",
+        required=True,
+    )
+    parser.add_argument(
+        "--input-file-type",
```

Comment:
I made another comment but I'll leave this here as well that we should add a similar arg for `--output-file-type`, you can default it to be the same as `--input-file-type`.

Reply:
Sorry for missing that earlier, resolved via : https://github.com/NVIDIA/NeMo-Curator/pull/58/commits/c6b5d9ed03dc790ab9b675b19de015478c686746